Cargo.toml

```diff
@@ -11,6 +11,9 @@ documentation = "https://docs.rs/goblin"
 categories = ["parsing", "development-tools::debugging"]
 include = ["src/**/*", "Cargo.toml", "CHANGELOG.md", "LICENSE", "README.md", "etc/*", "examples/*", "tests/*", "fuzz/**/*"]
 
+[dev-dependencies]
+failure = "0.1.3"
```

Comment:
I don’t want a dev dep for the example; various upstream deps have already broken our CI a few times and unwrap is sufficient.

Reply:
I removed it. However, could you be more detailed? This information could save me lots of hair in the future :)

Reply:
It was just becoming annoying to have deps for the sake of some small examples. Eg the rdr.rs file had env_logger dep, just because it made debugging it easier, but multiple times their downstream deps had upgraded their rustc and we didn’t so it broke our CI. It’s not that big a deal but I got tired of spurious failures from dependencies we don’t even use so I basically ripped them all out in a minor rage :)